package.json

```diff
@@ -8,8 +8,9 @@
     "tools/**/*"
   ],
   "scripts": {
-    "build": "lerna run build --stream",
+    "build": "lerna run build --stream && yarn build-blazor-app",
```

Comment:
How much time does the blazor app build add to our build command?

Reply:
About 6-10 seconds.